code/Modules/Props/Types/Prop.cs

```diff
@@ -11,12 +11,15 @@ namespace ssl.Modules.Props.Types
     /// </summary>
     public partial class Prop : ModelEntity, ISelectable
     {
+        public const string Tag = "Prop";
+        
```

Comment:
suggestion: PropTag

Reply:
Since we're in Prop class isn't useless to use "Prop.PropTag" ?
Prop.Tag is pretty straightforward imo (same for Item.Tag)

Reply:
Both will be refactored to extends the same object since Prop and Item have a lot of similarities